pkg/proxy/iptables/number_generated_rules_test.go

```diff
@@ -59,63 +59,63 @@ func TestNumberIptablesRules(t *testing.T) {
 			name:                "0 Services 0 EndpointsPerService - ClusterIP",
 			services:            0,
 			epPerService:        0,
-			expectedFilterRules: 4,
```

Comment:
the following are changed (decrement by 1)  as the drop rule is not written into `filterRules` buffer.